collectors/macos.plugin/multi_metadata.yaml

```diff
@@ -42,14 +48,208 @@ modules:
+              description: Enable or disable monitoring of ICMP neighbor metrics (I/O Solicits, I/O Advertisements).
+              default_value: auto
+              required: false
+            - name: icmp types
+              section_name: plugin:macos:sysctl
+              description: Enable or disable monitoring of ICMP types metrics (I/O Type1, I/O Type128, I/O Type129, Out Type133, Out Type135, In Type136, Out Type145).
+              default_value: auto
+              required: false
+            - name: space usage for all disks
+              section_name: plugin:macos:sysctl
+              description: Enable or disable monitoring of space usage for all disks metrics (available, used, reserved for root).
+              default_value: yes
+              required: false
+            - name: inodes usage for all disks
+              section_name: plugin:macos:sysctl
+              description: Enable or disable monitoring of inodes usage for all disks metrics (available, used, reserved for root).
+              default_value: yes
+              required: false
+            - name: bandwidth
+              section_name: plugin:macos:sysctl
+              description: Enable or disable monitoring of bandwidth metrics (received, sent).
+              default_value: yes
+              required: false
+            - name: system uptime
+              section_name: plugin:macos:sysctl
+              description: Enable or disable monitoring of system uptime metrics (uptime).
+              default_value: yes
+              required: false
+            - name: cpu utilization
+              section_name: plugin:macos:mach_smi
+              description: Enable or disable monitoring of CPU utilization metrics (user, nice, system, idel).
+              default_value: yes
+              required: false
+            - name: system ram
+              section_name: plugin:macos:mach_smi
+              description: Enable or disable monitoring of system RAM metrics (Active, Wired, throttled, compressor, inactive, purgeable, speculative, free).
+              default_value: yes
+              required: false
+            - name: swap i/o
+              section_name: plugin:macos:mach_smi
+              description: Enable or disable monitoring of SWAP I/O metrics (I/O Swap).
+              default_value: yes
+              required: false
+            - name: memory page faults
+              section_name: plugin:macos:mach_smi
+              description: Enable or disable monitoring of memory page faults metrics (memory, cow, I/O page, compress, decompress, zero fill, reactivate, purge).
+              default_value: yes
+              required: false
+            - name: disk i/o
+              section_name: plugin:macos:iokit
+              description: Enable or disable monitoring of disk I/O metrics (In, Out).
+              default_value: yes
+              required: false
         examples:
           folding:
             enabled: true
```

Comment:
All scopes have no description. There is boilerplate text for both global and other scopes, please take a look at python collectors and even charts.d, and add these in